controllers/installation_controller.go

```diff
@@ -121,10 +124,11 @@ func (r *InstallationReconciler) Reconcile(ctx context.Context, req ctrl.Request
 
 		// Nothing for us to do at this point
 		log.V(Log4Debug).Info("Reconciliation complete: A porter agent has already been dispatched.")
-		if r.PorterGRPCClient != nil {
-			return r.CheckOrCreateInstallationOutputsCR(ctx, log, inst)
+		r.PorterGRPCClient, err = createPorterGRPCClient()
+		if err != nil {
```

Comment:
Probably could debug log to show that we are not failing but not doing outputs because the grpc server is not available.